templates/patterns/date_block/date_block.ui_patterns.yml

```diff
@@ -0,0 +1,24 @@
+date_block:
+  label: "Date Block"
+  description: "Date block allows to present a date block that can be placed in other components."
+  variants:
+    default:
+      label: Default
+  fields:
+    day:
+      type: "number"
+      label: "Day"
+      preview: "27"
```

Comment:
We can add descriptions for all fields, so in the patterns example page is all more clear and is useful.

Reply:
I removed, that doesn't give more info, is the day number...